spec/features/recipes/show_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "rails_helper"
+
+RSpec.describe "the recipes show" do 
+  meal = Recipe.create!(name: "spicy tacos", complexity: 1, genre: "mexican")
+
+  item_1 = Ingredient.create!(name: "salt", cost: 1)
+  item_2 = Ingredient.create!(name: "pepper", cost: 2)
+
+  meal.ingredients << item_1 << item_2
+  # RecipeIngredient.create!(recipe_id: meal.id, ingredient_id: item_1.id, ingredient_id: item_1.id)
+
+  # User Story 2 - Recipes Show
+  it "lists the the recipes attributes and  needed compenents" do
+    # When I visit '/recipes/:id',
+   visit "/recipes/#{meal.id}"
+  # Then I see the recipe's name, complexity and genre,
+  expect(page).to have_content(meal.name)
+  expect(page).to have_content(meal.complexity)
+  expect(page).to have_content(meal.genre)
+  # and I see a list of the names of the ingredients for the recipe.
+    expect(page).to have_content(item_1.name)
+    expect(page).to have_content(item_2.name)
+    expect(page).to have_content(item_1.cost)
+    expect(page).to have_content(item_2.cost)
```

Comment:
Same idea here -- it's good to test for the presence of these attributes, but the assertions are not as meaningful unless we can verify that they're showing up in the right location. If you set up your view to display the ingredient and its cost side by side, you could have something like: `expect(page).to have_content("#{item_1.name}: #{item_1.cost}")`